tools/build_pytorch_libs.bat

```diff
@@ -212,7 +212,7 @@ goto:eof
                   -DTORCH_BUILD_VERSION="%PYTORCH_BUILD_VERSION%" ^
                   -DBUILD_TORCH="%BUILD_TORCH%" ^
                   -DNVTOOLEXT_HOME="%NVTOOLEXT_HOME%" ^
-                  -DNO_API=ON ^
+                  -DNO_API=OFF ^
```

Comment:
You can just remove this line instead I think.

Reply:
ok